modules/cli/src/main/scala/scala/cli/commands/shared/ProjectVersionOptions.scala

```diff
@@ -0,0 +1,19 @@
+package scala.cli.commands.shared
+
+import caseapp.*
+
+import scala.cli.commands.tags
+
+// format: off
+final case class ProjectVersionOptions(
+  @HelpMessage("Method used to compute the project version")
+  @ValueDescription("git|git:tag|command:...")
+  @Tag(tags.restricted)
+    computeVersion: Option[String] = None
```

Comment:
Move the `version` parameter from `PublishParamsOptions` into `ProjectVersionOptions`. I think it's also directly connected to the project version.